pkg/minikube/cluster/cluster.go

```diff
@@ -227,11 +228,53 @@ var assets = []fileToCopy{
 	},
 }
 
-func UpdateCluster(d drivers.Driver) error {
+func updateLocalkubeFromURL(config KubernetesConfig, client *ssh.Client) error {
+	resp := &http.Response{}
```

Comment:
Could this just be:

var resp http.Response

downloader := func() (err error) {
  resp, err := http.Get(...
}

to avoid the tmpResp shuffle.